tests/test_utils/test_patch_utils.py

```
@@ -868,15 +869,15 @@ def test_path_column_leave_extension(self, random_directory_spool):
     def test_mixed_path_sources_use_metadata(self, random_spool):
         """Mixed real and memory paths consistently use metadata names."""
         df = random_spool.get_contents().iloc[:2].copy()
-        df["path"] = ["/tmp/real_file.h5", "memory://registry/patch"]
+        df["source_path"] = ["/tmp/real_file.h5", "memory://registry/patch"]
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/real_file.h5"

(S108)